src/scripts/main.js

```diff
@@ -1,3 +1,320 @@
 'use strict';
 
-// write your code here
+const initialField = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
+let upDatefield = replaceRandomZero(initialField);
+let scoreNumber = 0;
+const cells = document.getElementsByClassName('field-cell');
+const scoreText = document.getElementsByClassName('game-score');
+const buttonStart = document.querySelector('button');
+const startText = document.getElementsByClassName('message-start');
+
+buttonStart.addEventListener('click', () => {
+  scoreNumber = 0;
+  upDatefield = initialField;
+  upDatefield = replaceRandomZero(upDatefield);
+  initial(upDatefield);
+  styleCells();
+
+  startText[0].style.display = 'none';
+  buttonStart.innerHTML = 'restart';
+  buttonStart.style.backgroundColor = 'rgba(254,105,110,255)';
+
+  document.addEventListener('keydown', function() {
+    switch (event.key) {
+      case 'ArrowLeft':
+        upDatefield = keyLeft(upDatefield);
+        upDatefield = replaceRandomZero(upDatefield);
+        initial(upDatefield);
+        styleCells();
+        checker();
```

Comment:
you repeat this logic in each case, move it out of the switch case and write only one time